src/main/java/com/tabgok/entity/Entity.java

```diff
@@ -0,0 +1,7 @@
+package com.tabgok.entity;
+
+import com.tabgok.harvester.commands.CommandResult;
+
+
```

Comment:
I think I am implementing something similar to futures...but I don't have enough experience with those to know.  A "command" is a generic "executed something" and a CommandResult is a "executed something's return value"